lighthouse-core/gather/gatherers/manifest.js

```diff
@@ -36,6 +37,12 @@ class Manifest extends Gatherer {
   afterPass(options) {
     return options.driver.getAppManifest()
       .then(response => {
+        const isBomEncoded = Buffer.from(response.data).slice(0, BOM_ENCODING.length)
+          .equals(new Buffer(BOM_ENCODING));
```

Comment:
what's the compatibility like if we skip the Buffer slicing for the check (so files without a BOM don't have to go through that) and do `const isBomEncoded = response.data.charCodeAt(0) === 65279;`. Does that work cross platform?